src/index.js

```diff
@@ -0,0 +1,57 @@
+import './styles/style.css';
+
+import mainPage from './tabs/main-page';
+import menuPage from './tabs/menu-page';
+import infoPage from './tabs/info-page';
+
+function component() {
+  const content = document.getElementById('content');
```

Comment:
__[Optional]__ It is a requirement for this project that you use ES6 syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/)). Make use of the `arrow` function in this case.